java/test/android/controller/src/main/java/com/zeroc/testcontroller/ControllerActivity.java

```diff
@@ -126,7 +128,18 @@ public void onNothingSelected(AdapterView<?> arg0)
                 }
             });
         s.setSelection(0);
-        app.startController(this, bluetooth);
+
+        // Start the controller in a background thread. Starting the controller creates the ObjectAdapter which makes
+        // IO calls. Android doesn't allow  making IO calls from the main thread.
```

Comment:
An extra space snuck in here.
```suggestion
        // IO calls. Android doesn't allow making IO calls from the main thread.
```